backend-flask/README.txt

```diff
@@ -0,0 +1,25 @@
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
+
+This installs the venv package and names the folder "venv".
```

Comment:
If you don't want people to commit their `venv` folders, you can create a `.gitignore` file and add the folder to it. VSCode gives some helpful highlighting in its folder tree to indicate what is ignored when committing files; I do not know if Atom has a similar feature, but it likely does.

Reply:
Done.